app/src/main/kotlin/org/eclipse/kuksa/testapp/databroker/viewmodel/OutputViewModel.kt

```diff
@@ -27,18 +32,23 @@ import java.time.format.DateTimeFormatter
 private const val MAX_NUMBER_LOG_ENTRIES = 100
 
 class OutputViewModel : ViewModel() {
+    private val handler = Handler(Looper.getMainLooper())
```

Comment:
Can we use `viewModelScope.launch {  }` here? ViewModel should come with the support to dispatch to the main loop out of the box.

Reply:
Good find, I forgot about that one and searched like an idiot for some "runOnUiThread" method in viewModel.